cellengine/resources/experiment.py

```diff
@@ -274,7 +381,7 @@ def get_statistics(
         )
 
     @property
-    def scalesets(self) -> List[ScaleSet]:
+    def scalesets(self) -> ScaleSet:
         """List all scalesets in the experiment."""
```

Comment:
The doc string, return type and method name don't quite match up now. I'd go all singular: `def scaleset` and `Gets the experiment's ScaleSet`.

Reply:
:+1: